server/src/main/java/org/elasticsearch/cluster/metadata/IndexNameExpressionResolver.java

```diff
@@ -158,9 +158,16 @@
             expressions = expressionResolver.resolve(context, expressions);
         }
 
+        final String noIndicesErrorMessage = "no indices exist";
```

Comment:
I don't think this should be a variable: either a constant (private static final) or just duplicated where the two exceptions are thrown, up to you.

Reply:
I duplicated the message where the exceptions are thrown